csharp/ql/lib/semmle/code/csharp/Location.qll

```diff
@@ -50,6 +50,27 @@ class Location extends @location {
 
   /** Gets the 1-based column number (inclusive) where this location ends. */
   final int getEndColumn() { this.hasLocationInfo(_, _, _, _, result) }
+
+  /** Holds if this location starts strictly before the specified location. */
+  bindingset[this, other]
```

Comment:
Aha, so it is possible to declare bindingsets for member predicates?
Or is the bindingset just declared because you want to use the `inline_late` pragma?

Reply:
Yes, it's needed for `inline_late`.